### DISCUSSION TITLE
ESP32 DevkitC Compatibility

### DISCUSSION THREAD

**Discussion by domwolfy** [adafruit/Adafruit_TinyUSB_Arduino#305] [Q&A]
On compiling error: 'Adafruit_USBD_MIDI' does not name a type; did you mean 'Adafruit_USBD_CDC'?

Screenshots
I'd like to know if this board is actually incompatibile with this library?
I know the board doesn't have USB native, I thought this library with TinyUSB was to overcome that problem?
The readme lists esp32s in compatible boards, if this board is an exception, what makes it unsupportable?
some esp32 boards will work but not others should the readme not clarify this?

**Comment by hathach** [comment]
USB OTG such as in S2/S3 is required to use tinyusb. It is too obvious to mention
